sdk/shared/src/main/scala/org/ergoplatform/sdk/BlockchainParameters.scala

```diff
@@ -0,0 +1,52 @@
+package org.ergoplatform.sdk
+
+/** Blockchain parameters re-adjustable via miners voting and voting-related data.
+  * All these fields are included into extension section of a first block of a voting epoch.
+  *
+  * @param storageFeeFactor cost of storing 1 byte in UTXO for four years, in nanoErgs
+  * @param minValuePerByte cost of a transaction output, in computation unit
+  * @param maxBlockSize max block size, in bytes
+  * @param tokenAccessCost cost of a token contained in a transaction, in computation unit
+  * @param inputCost cost of a transaction input, in computation unit
+  * @param dataInputCost cost of a transaction data input, in computation unit
+  * @param outputCost cost of a transaction output, in computation unit
+  * @param maxBlockCost computation units limit per block
+  * @param softForkStartingHeight height when voting for a soft-fork had been started
+  * @param softForkVotesCollected votes for soft-fork collected in previous epochs
+  * @param blockVersion Protocol version activated on the network
+  */
+case class BlockchainParameters(
```

Comment:
Why not ErgoParameters ? Parameters are not related to a generic blockchain, rather, the Ergo one (I guess it was named ErgoLikeParameters to support Ergo-like blockchains as well)

Reply:
The new name clarifies that it is `blockchain` parameters (those stored in blockchain), not some other parameters.
The package `org.ergoplatform.sdk` already suggests it is related to Ergo blockchain, so no need to add Ergo once again.

`ErgoLike` prefix is pretty useless (ErgoLikeContext doesn't tell anything about what kind of context it is, InterpreterContext is much better).

I was thinking to rename all `ErgoLike` names at some point as part of further refactoring and code simplification

Reply:
So for other Ergo-like blockchains (e.g. sidechains) you will have different packages?

Reply:
Yes, probably. Everything in org.ergoplatform is part of reference implementation. 
It will depend on how other projects will be architected, as a layer on top of core ergo packages, or as a repo fork with the necessary changes.